autorest/azure/token.go

```diff
@@ -1,12 +1,19 @@
 package azure
 
 import (
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/sha1"
+	"encoding/base64"
+	"fmt"
 	"net/http"
 	"net/url"
 	"strconv"
 	"time"
 
 	"github.com/Azure/go-autorest/autorest"
+	jwt "github.com/dgrijalva/jwt-go"
+	pkcs12 "golang.org/x/crypto/pkcs12"
```

Comment:
I'm not a huge fan of unnecessarily naming imports. It's not a pattern I've seen broadly used. It is intended as an escape value when there are unavoidable conflicts.

Also, is "jwt" necessary? One example (see https://github.com/someone1/gcp-jwt-go) uses "jwt" directly without renaming the import. Also, the package name is "jwt".